### PR TITLE
Feature/wb 24 create world

### PR DESCRIPTION
**Pull request by terbooter**

### CODE REVIEW

File: internal/config/common.go [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package config
 
 type Common struct {
-	IntrospectURL string `yaml:"introspect_url" envconfig:"BACKEND_INTROSPECT_URL"`
+	IntrospectURL   string `yaml:"introspect_url" envconfig:"BACKEND_INTROSPECT_URL"`
+	MediaManagerURL string `yaml:"media_manager_url" envconfig:"RENDER_SERVICE_URL"`
```
Comment (cnaize) [inline internal/config/common.go:5]: why not MEDIA_MANAGER_URL?

File: internal/world_builder/world_builder.go [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	AssetParameters            string      `json:"asset_parameters"`
+	Parameters2D               string      `json:"parameters2D"`
+	Parameters3D               string      `json:"parameters3D"`
+	Secret                     int         `json:"secret"`
+	FrameTemplates             interface{} `json:"frame_templates"`
+	AllowedSubspaces           interface{} `json:"allowed_subspaces"`
+	ChildPlacement             interface{} `json:"child_placement"`
+	Minimap                    int         `json:"minimap"`
+	Visible                    int         `json:"visible"`
+	Metadata                   interface{} `json:"metadata"`
```
Comment (cnaize) [inline internal/world_builder/world_builder.go:39]: is it WorldMeta? or why interface{} is here?

File: internal/world_builder/world_builder.go [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+		Fontcolor: [4]int{220, 220, 200, 0},
+		Wrap:      false,
+		PadX:      0,
+		PadY:      1,
+		AlignH:    "center",
+		AlignV:    "center",
+	},
+}
+
+var getMainDomain = `SELECT value FROM node_settings WHERE name = 'main_domain'`
```
Comment (cnaize) [inline internal/world_builder/world_builder.go:103]: it's better to put it in database layer

File: internal/world_builder/world_builder.go [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+	err = w.insertRows(meta, template)
+	if err != nil {
+		return meta, errors.WithMessage(err, "failed to insertRows")
+	}
+
+	return meta, nil
+}
+
+func (w *WorldBuilder) insertRows(meta *WorldMeta, t *WorldTemplate) error {
```
Comment (cnaize) [inline internal/world_builder/world_builder.go:150]: it's better to put it in database layer

File: internal/config/common.go (hunk 1 above)
Comment (terbooter) [reply internal/config/common.go:5]: `RENDER_SERVICE_URL ` already existing env var
need to update all deployment to change it

File: internal/world_builder/world_builder.go (hunk 2 above)
Comment (terbooter) [reply internal/world_builder/world_builder.go:39]: `WorldTemplate.Metadata` will be stored in DB space.metadata column
`WorldMeta` struct will return to client in http response

File: internal/world_builder/world_builder.go (hunk 3 above)
Comment (terbooter) [reply internal/world_builder/world_builder.go]: Agreed, DB functions moved to DB layer

File: internal/world_builder/world_builder.go (hunk 4 above)
Comment (terbooter) [reply internal/world_builder/world_builder.go]: Done

File: internal/world_builder/world_builder.go [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+	mediaManagerOptions.Text.String = text
+
+	jsonStr, err := json.Marshal(mediaManagerOptions)
+	if err != nil {
+		return "", errors.WithMessage(err, "failed to marshal renderer options")
+	}
+
+	url := fmt.Sprintf("%s/render/addframe", w.config.Common.MediaManagerURL)
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(jsonStr))
```
[the remark this answers is not in the document]
Comment (terbooter) [reply internal/world_builder/world_builder.go:185]: Added error check

File: internal/world_builder/world_builder.go [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+	Secret                     int         `json:"secret"`
+	FrameTemplates             interface{} `json:"frame_templates"`
+	AllowedSubspaces           interface{} `json:"allowed_subspaces"`
+	ChildPlacement             interface{} `json:"child_placement"`
+	Minimap                    int         `json:"minimap"`
+	Visible                    int         `json:"visible"`
+	Metadata                   interface{} `json:"metadata"`
+	Position                   interface{} `json:"position"`
+	Asset                      interface{} `json:"asset"`
+	UniqueAttributeInSpaceType interface{} `json:"uniqueAttributeInSpaceType"`
```
Comment (cnaize) [inline internal/world_builder/world_builder.go:40]: why here are interfaces? it's better to define appropriate structures here for easy further modifications

**Pull request merged**